'make docs' is broken on master

```
Exception occurred:
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/docutils/writers/_html_base.py", line 673, in depart_document
    assert not self.context, 'len(context) = %s' % len(self.context)
AssertionError: len(context) = 6
The full traceback has been saved in /var/folders/zr/t20pv25j52jdd5_r0fyp65rsz3882v/T/sphinx-err-IDOxGd.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

longer form.

```
# Sphinx version: 1.5
# Python version: 2.7.11 (CPython)
# Docutils version: 0.13.1 release
# Jinja2 version: 2.8
# Last messages:
#   /Users/robj/workspace/paasta_tools/docs/source/deploy_marathon_services.rst:: WARNING: document isn't included in any toctree
#   /Users/robj/workspace/paasta_tools/docs/source/installation/example_cluster.rst:: WARNING: document isn't included in any toctree
#   /Users/robj/workspace/paasta_tools/docs/source/monitoring/index.rst:: WARNING: document isn't included in any toctree
#   done
#   preparing documents...
#   done
#   writing output... [  0%] about/contract
#   writing output... [  1%] about/glossary
#   writing output... [  2%] about/paasta_principles
#   writing output... [  2%] about/smartstack_interaction
# Loaded extensions:
#   sphinx.domains.cpp (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/domains/cpp.pyc
#   sphinx.builders.applehelp (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/applehelp.pyc
#   sphinx.directives (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/directives/__init__.pyc
#   sphinx.builders.changes (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/changes.pyc
#   sphinx.builders.websupport (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/websupport.pyc
#   sphinx.domains.c (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/domains/c.pyc
#   sphinx.builders.dummy (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/dummy.pyc
#   sphinx.builders.texinfo (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/texinfo.pyc
#   sphinx.builders.epub (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/epub.pyc
#   sphinx.builders.linkcheck (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/linkcheck.pyc
#   sphinx.domains.std (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/domains/std.pyc
#   sphinx.directives.other (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/directives/other.pyc
#   sphinx.builders.epub3 (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/epub3.pyc
#   sphinx.domains.javascript (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/domains/javascript.pyc
#   sphinx.directives.code (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/directives/code.pyc
#   sphinx.ext.autodoc (1.5) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/ext/autodoc.pyc
#   alabaster (0.7.9) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/alabaster/__init__.pyc
#   sphinx.builders.text (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/text.pyc
#   sphinx.builders.qthelp (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/qthelp.pyc
#   sphinx.roles (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/roles.pyc
#   sphinx.builders.htmlhelp (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/htmlhelp.pyc
#   sphinx.directives.patches (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/directives/patches.pyc
#   sphinx.builders.latex (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/latex.pyc
#   sphinx.builders.html (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/html.pyc
#   sphinx.builders.devhelp (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/devhelp.pyc
#   sphinxcontrib.programoutput (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinxcontrib/programoutput.pyc
#   sphinx.builders.gettext (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/gettext.pyc
#   sphinx.builders.xml (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/xml.pyc
#   sphinx.builders.manpage (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/manpage.pyc
#   sphinx.domains.rst (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/domains/rst.pyc
#   sphinx.domains.python (unknown version) from /Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/domains/python.pyc
Traceback (most recent call last):
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/cmdline.py", line 296, in main
    app.build(opts.force_all, filenames)
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/application.py", line 333, in build
    self.builder.build_update()
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 251, in build_update
    'out of date' % len(to_build))
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 322, in build
    self.write(docnames, list(updated_docnames), method)
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 360, in write
    self._write_serial(sorted(docnames), warnings)
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 368, in _write_serial
    self.write_doc(docname, doctree)
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/builders/html.py", line 452, in write_doc
    self.docwriter.write(doctree, destination)
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/docutils/writers/__init__.py", line 80, in write
    self.translate()
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/sphinx/writers/html.py", line 48, in translate
    self.document.walkabout(visitor)
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/docutils/nodes.py", line 187, in walkabout
    visitor.dispatch_departure(self)
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/docutils/nodes.py", line 1895, in dispatch_departure
    return method(node)
  File "/Users/robj/workspace/paasta_tools/.tox/docs/lib/python2.7/site-packages/docutils/writers/_html_base.py", line 673, in depart_document
    assert not self.context, 'len(context) = %s' % len(self.context)
AssertionError: len(context) = 6
```

so it's the about/smartstack_interaction file it's upset about

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,4 +1,5 @@
 -r requirements.txt
+docutils==0.12
 flake8==2.5.0
 git+git://github.com/solarkennedy/sphinxcontrib-programoutput#egg=package-two
 mock==2.0.0

